USERGUIDE.md

```diff
@@ -723,6 +724,17 @@ And the Mini App will be loaded like the following scheme,
 
 ```mscheme.rakuten//miniapp/index.html?param1=value1&param2=value2```
 
+
+<a id="permissions-from-host-app"></a>
+
+### Permissions required from the Host app
+
+Mini app requires following set of Device permissions that needed to be added in the Info.plist,
```

Comment:
Mini App SDK requires the host app to include the following set of device permissions into its Info.plist file:

Reply:
Done